README.md

```diff
@@ -45,6 +45,9 @@ custom:
     # (optional) Buckets to remove before a stack is deployed.
     bucketsToCleanOnDeploy:
       - oldBucketName
+
+    # (optional) Get deployment bucket name from AWS. Works only for `sls s3remove` and `sls remove`  
+    autoResolve: false
```

Comment:
The plugin is not meant to clean *just* the Serverless deployment buckets, think of stacks that create additional S3 buckets for other purposes, such as web apps. I think `autoResolve` in this context is a bit too open-ended, so I'd suggest a more targeted name. E.g. `cleanDeploymentBucket`, `includeDeploymentBucket`, or something similar?

Reply:
@f00f sorry for mentioning ) just because you are appears in "maintainers" 

@coyoteecd good point. For renaming option.